openid-4-verifiable-presentations-1_0.md

```diff
@@ -461,9 +491,9 @@ Location: https://client.example.org/universal-link?
   8%22%5D%7D%7D%7D
 ```
 
-* `entity_id`: This value indicates that the Client Identifier is an Entity Identifier defined in OpenID Federation [@!OpenID.Federation]. Processing rules given in [@!OpenID.Federation] MUST be followed. Automatic Registration as defined in [@!OpenID.Federation] MUST be used. The Authorization Request MAY also contain a `trust_chain` parameter. The final Verifier metadata is obtained from the Trust Chain after applying the policies, according to [@!OpenID.Federation]. The `client_metadata` parameter, if present in the Authorization Request, MUST be ignored when this Client Identifier scheme is used.
+* `https`: This value indicates that the Client Identifier is an Entity Identifier defined in OpenID Federation [@!OpenID.Federation]. Since the Entity Identifier is already defined to start with `https:`, this Client Identifier Scheme MUST NOT be prefixed additionally. Processing rules given in [@!OpenID.Federation] MUST be followed. Automatic Registration as defined in [@!OpenID.Federation] MUST be used. The Authorization Request MAY also contain a `trust_chain` parameter. The final Verifier metadata is obtained from the Trust Chain after applying the policies, according to [@!OpenID.Federation]. The `client_metadata` parameter, if present in the Authorization Request, MUST be ignored when this Client Identifier scheme is used. Example Client Identifier: `https://federation-verifier.example.com`.
```

Comment:
Following Brian's comments, I was wondering if we should open this up to other client ID schemes using `https` without an additional prefix. For example:

```suggestion
* `https`: This value indicates that the Client Identifier is an HTTPS URL. Since the Entity Identifier is already defined to start with `https:`, this Client Identifier Scheme MUST NOT be prefixed additionally. Unless another mechanism for resolving the Client Identifier and the metadata is defined within a profile of this specification, the following rules apply: 
     * The Client Identifier MUST be treated as an OpenID Federation [@!OpenID.Federation] Entity Identifier. Processing rules given in [@!OpenID.Federation] MUST be followed. Automatic Registration as defined in [@!OpenID.Federation] MUST be used. The Authorization Request MAY also contain a `trust_chain` parameter. The final Verifier metadata is obtained from the Trust Chain after applying the policies, according to [@!OpenID.Federation]. The `client_metadata` parameter, if present in the Authorization Request, MUST be ignored when this Client Identifier scheme is used. Example Client Identifier: `https://federation-verifier.example.com`.
```

Reply:
that's like saying "here are the rules we define for prefixing, but other rules for the same prefix can be defined outside this specification", so I don't think that's a good idea.

Reply:
I agree with Kristina, I'm reluctant to open the door to allowing other schemes to use https without prefixing. It just creates a mess and confusion and interoperability problems.

I'm only reluctantly accepting that Federation should be allowed to use https urls without a prefix. I cannot see any strong reason to let others do so. Having a prefix really causes no technical issue unless something is already in production, and I'm pretty sure federation is the only spec that can even attempt the 'already in production' argument.